src/RDBViewProviders.ts

```diff
@@ -1,19 +1,25 @@
 import * as vscode from 'vscode';
+import * as path from 'path';
```

Comment:
Since the rest of the project uses spaces, can we standardize on spaces for spacing